workflows/offline_ml_diags/README.md

```diff
@@ -19,48 +19,10 @@ python -m offline_ml_diags.compute_diags \
 ```
```

Comment:
Thanks for updating. It would be nice if we had autodocs for `offline_ml_diags` but that's out of the scope of this PR, maybe make an issue if you like